src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DocumentHighlightHandler.cs

```diff
@@ -66,14 +77,21 @@ public async Task<DocumentHighlight[]> HandleRequestAsync(DocumentHighlightParam
                 throw new ArgumentNullException(nameof(clientCapabilities));
             }
 
+            _logger.LogInformation($"Starting request for {request.TextDocument.Uri}.");
+
             if (!_documentManager.TryGetDocument(request.TextDocument.Uri, out var documentSnapshot))
             {
+                _logger.LogWarning($"Failed to find document {request.TextDocument.Uri}.");
```

Comment:
I assume the Logger with name "DocumentHightlightHandler" handles some context here? Otherwise we need to be more detailed about what request failed to find the document.

Reply:
Yep! There's a couple of layers here:
1. On the activity trace, each activity (ie. `textDocument/highlight`) request is broken down seperately (see screenshot in summary)
2. We prepend all the logs with the handler name (`DocumentHighlightHandler`)

So this would look like:

```
[DocumentHighlightHandler] Failed to find document {request.TextDocument.Uri}.
```